### DISCUSSION TITLE
NLL of UCI datasets

### DISCUSSION THREAD

**Discussion by evwalz** [stanfordmlgroup/ngboost#279] [Q&A]
Hello everyone,
I would like to reproduce Table 1 in the NGBoost paper showing the negative log score (NLL) for 10 benchmark datasets from UCI. In the directory 'examples/experiments/regression_exp.py' the corresponding file is provided. I set the parameters as described in the paper. However, I am having difficulties in getting similar values for the negative log score. Am I missing something? Do I need to tune additional parameters during training to end up with similar NLL values?

**Comment by alejandroschuler** [comment]
Hmm, good question. @avati or @tonyduan might have an answer.
